src/js/lib/ajax.js

```diff
@@ -101,6 +103,8 @@ var ajax = function(opt, success, failure) {
       try {
         if (opt.type === 'json') {
           body = JSON.parse(body);
+        } else if (op.type === 'xml') {
```

Comment:
You can object the req object with a handler `function (data, status, req)` and operate on req.responseXML. The issue however is that this is android specific.